### PR TITLE
removed staging/src/k8s.io/client-go/tools/leaderelection

### PR DESCRIPTION
**Pull request by siddsuresh97**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by siddsuresh97** [comment]
Signed the CLA

**Comment by mattjmcnaughton** [comment]
@siddsuresh97 thanks for your pr!

Could you please fill in the pull request template? Specifically, it would be great to have context around why this pr is necessary, and you'll also need a release note before its possible to merge.

Thanks so much :)

File: staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go [hunk 1]
```diff
@@ -98,7 +99,8 @@ func NewLeaderElector(lec LeaderElectionConfig) (*LeaderElector, error) {
 	return &le, nil
 }
 
-type LeaderElectionConfig struct {
+//Config structure for leader election.
+type Config struct {
```
Comment (caesarxuchao) [inline staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go]: Config is a better name, but I don't think it's worth a breaking change for all client-go users.

**Comment by fedebongio** [comment]
cc @cheftako

**Comment by siddsuresh97** [comment]
@mattjmcnaughton . 

Which issue this PR fixes : fixes a part of 68026

**Comment by siddsuresh97** [comment]
@caesarxuchao there was an error while running hack/verify-golint.sh as mentioned in https://github.com/kubernetes/kubernetes/issues/68026. I changed the name to fix this error.

File: staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go (hunk 1 above)
Comment (siddsuresh97) [reply staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go]: @caesarxuchao there was an error while running hack/verify-golint.sh as mentioned in #68026. I changed the name to fix this error.

**Pull request closed without merging**